layers/+lang/typescript/README.org

```diff
@@ -27,12 +27,15 @@ This layer provides:
 - Imenu
 - linting
 - tsx mode
+- formatting
 
 * Install
 ** Pre-requisites
 You will need =node.js v0.12.0= or greater
 
-If you want linting run:  =npm instell -g typescript=  =npm install -g tslint=  
+If you want linting run:  =npm install -g typescript=  =npm install -g tslint=  
```

Comment:
I don't really know `npm` but should these two snippets be grouped in one, like `npm install -g typescript tslint`?